components/Profile/ExerciseChart.jsx

```diff
@@ -103,7 +104,7 @@ export default () => {
       { loading
         ? (
           <SpinnerContainer>
-            <ActivityIndicator color={COLORS[0]} size="large" />
+            <ActivityIndicator color={COLORS[colorTheme][0]} size="large" />
```

Comment:
It's not letting me comment on the actual button, but clicking the "Select Exercises" causes an invalid hook error.